content.js

```diff
@@ -0,0 +1,13 @@
+function checkForOnionMeta() {
+    const meta = document.querySelector('meta[http-equiv="Onion-Location"]');
+    if (meta && meta.content) {
+        return meta.content;
+    }
+    return null;
+}
+
+const onionLocation = checkForOnionMeta();
```

Comment:
At what point does this code run? After the document if fully loaded?

Reply:
I probably sound stupid, but I'm not too sure. I looked at how other extensions check for metatags in the DOM and copied their implementation. I sent this PR with the hopes that someone with more experience than me could look over it and fix any of my oversights.